packages/strapi-parts/src/Layout/__tests__/HeaderLayout.e2e.js

```diff
@@ -22,10 +22,36 @@ describe('HeaderLayout', () => {
   describe('sticky', () => {
     it('triggers axe on the document', async () => {
       await page.goto(
-        'http://localhost:6006/iframe.html?id=design-system-layouts-headerlayouts--sticky&viewMode=story',
+        'http://localhost:6006/iframe.html?id=design-system-layouts-headerlayout--sticky&args=&viewMode=story',
       );
       await injectAxe(page);
       await checkA11y(page);
     });
   });
+
+  describe('combined w/ scroll', () => {
+    it('triggers axe on the document', async () => {
+      await page.goto(
+        'http://localhost:6006/iframe.html?id=design-system-layouts-headerlayout--combined-w-scroll&args=&viewMode=story',
+      );
+      await injectAxe(page);
+      await checkA11y(page);
+    });
+
+    it('displays the sticky header when scrolling', async () => {
```

Comment:
I think it could be interesting to create the opposite of this test: make sure that the "base" header layout is well displayed when scrolling in the other sense. For now, we're only checking that the sticky is shown when scrolling down, but what happens when scrolling up? 😄